.github/workflows/ci.yml

```diff
@@ -71,10 +71,10 @@ jobs:
     strategy:
       matrix:
         amiga:
-        # - { name: "AmigaOS 3", host: "m68k-amigaos", system: "amigaos3" }
-        - { name: "AmigaOS 4", host: "ppc-amigaos", system: "amiga" }
+        - { name: "AmigaOS 3", host: "m68k-amigaos" }
+        - { name: "AmigaOS 4", host: "ppc-amigaos" }
 
     steps:
     - uses: actions/checkout@v3
     - name: Build
-      run: make SYSTEM=${{ matrix.amiga.system }} CROSS=${{ matrix.amiga.host }}-
+      run: make SYSTEM=amiga CROSS=${{ matrix.amiga.host }}-
```

Comment:
Can you add an artifact-upload here?  Something like:
```
      - uses: actions/upload-artifact@v2
        with:
          name: arcem-${{ matrix.amiga.host }}
          path: ./arcem
```

I can more easily test the executable then.  It'll be useful for the other builds too.

Reply:
OK, here you go: https://github.com/ArcEm-emu/ArcEm/suites/18855194594/artifacts/1100085343